seed/views/v3/users.py

```diff
@@ -123,7 +123,8 @@ def __init__(self, *args):
                     name='New User Fields',
                     required=True,
                     description="An object containing meta data for a new user: "
-                                "required - [last_name, role(viewer/owner/member), email]",
+                                "- Required - first_name, last_name, email "
```

Comment:
I prefer the "\n" explicitly added in. Any reason you took it out?

Reply:
just having so many issues with Travis today I thought it was causing my build to fail.  I will change it back.

Reply:
This is basically good to go, but think you can add the `\n` in this description? as well as the ones on L142? Sorry 🙏

Reply:
Ah, no that is my bad I thought I added them back.